ENVIRONMENTVARS.md

```diff
@@ -0,0 +1,59 @@
+# Research Catalog Environment Variables
+
+## Table of Contents
+
+- [General Information](#general-information)
+- [Server and Application-wide Variables](#server-and-application-wide-variables)
+- [Application Variables](#application-variables)
+- [Testing](#testing)
+- [AWS Elastic Beanstalk Environment Variables](#aws-elastic-beanstalk-environment-variables)
+
+## General Information
+
+Environment variables are used in this code repository to control how the application builds, how and where data is fetched for separate sections in the application, for rendering certain features, and for controlling data flow.
+
+General environment variables are declared in the `.env.example` file. A copy of this file should be made and saved as `.env.local` where real values should be added.
+
+Generally, environment variables are meant to be read through the `process.env` object _on the server_. Variables intended for use on the client side should be prefaced with NEXT_PUBLIC_ per Next's [docs](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables).
+
+If an environment variable is updated, make sure to restart the server for the application to pick up the new value.
+
+## Server and Application-wide Variables
+
+These variables are used to configure server settings and application-wide settings.
+
+## Application Variables
```

Comment:
Seems like "Application-wide" and "Application" variables are very similar or the same. Maybe consolidate into one section.

Reply:
Good call, fixed.